source/store.go

```diff
@@ -139,6 +143,14 @@ func (p *SingletonClientGenerator) ContourClient() (contour.Interface, error) {
 	return p.contourClient, err
 }
 
+func (p *SingletonClientGenerator) OpenShiftClient() (openshift.Interface, error) {
```

Comment:
Please add documentation for this method.

Reply:
Copied and modified comment from ContourClient